docs/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,49 +1,54 @@
 - What user actions should my reviewer step through to review this PR?
 - Which storybook stories should be reviewed?
 - Are there plans for follow up PR’s to further fix this bug or develop this feature?
 
-
-# Review Checklist
+# Checklist
+_PR Creator - Please cater the checklist to fit the review needed for your code changes._
+_PR Reviewer - Use the checklist during your review. Each point should be checkmarked or discussed before PR approval._
 
 ## General
-
-- [ ] Are the tests passing locally and on Travis?
-- [ ] Is the documentation up to date?
-
-## Components
-- [ ] Has a storybook story been created or updated?
-- [ ] Is the component accessible? 
-  - [ ] Can it be used with a screen reader? [BBC guide to testing with VoiceOver](https://bbc.github.io/accessibility-news-and-you/accessibility-and-testing-with-voiceover-os.html)
-  - [ ] Can it be used from the keyboard? [WebAIM guide to keyboard testing](https://webaim.org/techniques/keyboard/#testing)
-  - [ ] Is it passing accessibility checks in the storybook?
-
-## Apps
-
-- [ ] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
-- [ ] Does it work on mobile?
-- [ ] Can you `yarn panic && yarn bootstrap` or `docker-compose up --build` and app works as expected?
-
-## Publishing
-
-- [ ] Is the changelog updated?
-- [ ] Are the dependencies updated for apps and libraries that are using the newly published library?
-
-## Post-merging
-
-- [ ] Did the app deploy to https://frontend.preview.zooniverse.org/projects/:project-name/:owner or https://frontend.preview.zooniverse.org/about?
-- [ ] Is the new feature working or bug now fixed?
-  - [ ] Is there a Talk or blog post written to announce the new feature(s)?
-- [ ] Is the design working across browsers (Firefox, Chrome, Edge, Safari) and mobile?
-  - [ ] Is this approved by our designer?
-- [ ] Is this ready for production deployment?
+- [ ] Tests are passing locally and on Github.
+- [ ] Documentation is up to date. Changelog has been updated if appropriate.
+- [ ] You can `yarn panic && yarn bootstrap` or `docker-compose up --build` and FEM works as expected.
+- [ ] FEM works in all major desktop browsers: Firefox, Chrome, Edge, Safari.
+- [ ] FEM works in a mobile browser.
+
+## General UX
+- [ ] All pages of a FEM project load: Home Page, Classify Page, and About Pages
+- [ ] Can submit a classification
+- [ ] Can sign-in and sign-out
+- Example Staging Projects
+    - [i-fancy-cats](https://local.zooniverse.org:3000/projects/brooke/i-fancy-cats)
+    - [Transformers](https://local.zooniverse.org:3000/projects/darkeshard/test-project-2022)
+
+## Bug Fix
+- [ ] The PR creator has listed user actions to use when testing if bug is fixed.
+- [ ] The bug is fixed.
+- [ ] Unit tests are added or updated.
+
+## New Feature
+- [ ] The PR creator has listed user actions to use when testing the new feature.
+- [ ] Unit tests are included for the new feature.
+- [ ] A storybook story been created or updated.
```

Comment:
Should we also encourage documenting the new component on its Storybook Docs page?

Reply:
```suggestion
- [ ] A storybook story has been created or updated.
```
Small typo here.